lib/openstudio-standards/standards/Standards.AirLoopHVAC.rb

```diff
@@ -1805,8 +1805,12 @@ def air_loop_hvac_enable_multizone_vav_optimization(air_loop_hvac)
       oa_system = air_loop_hvac.airLoopHVACOutdoorAirSystem.get
       controller_oa = oa_system.getControllerOutdoorAir
       controller_mv = controller_oa.controllerMechanicalVentilation
-      controller_mv.setSystemOutdoorAirMethod('VentilationRateProcedure')
-      # Change the min flow rate in the controller outdoor air
+      if air_loop_hvac.model.version < OpenStudio::VersionString.new('3.3.0')
+        controller_mv.setSystemOutdoorAirMethod('VentilationRateProcedure')
+      else
+        controller_mv.setSystemOutdoorAirMethod('Standard62.1VentilationRateProcedure')
```

Comment:
I think we might want to consider using `62.1VentilationRateProcedureWithLimit` here. That's the intent in ASHRAE 90.1.

Reply:
Yeah I agree, didn't see this additional enumeration.  I will update branch.  Also, this is not urgent to merge, take your time and look through the regression tests.